### DISCUSSION TITLE
Build Error - Nextra [ loader.mjs]

### DISCUSSION THREAD

**Discussion by lucastey** [vercel/community#2460] [Help] [answered]
### Summary

I've been stuck on the following build error for a day or so, tried to recompile and yarn install and tweak the path variable but nothing is working.

yarn run v1.22.19
warning ../../../package.json: No license field
$ node ./scripts/gen-rss.js && next build
info  - Checking validity of types  
info  - Creating an optimized production build .node:internal/errors:465
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:372:5)
    at validateString (node:internal/validators:120:11)
    at Object.join (node:path:1172:7)
    at file:///Users/lucastay/Desktop/GitHub/Cavern_Working/node_modules/nextra/dist/loader.mjs:17:27
    at ModuleJob.run (node:internal/modules/esm/module_job:198:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:385:24)
    at async importModuleDynamicallyWrapper (node:internal/vm/module:437:15) {
  code: 'ERR_INVALID_ARG_TYPE'
}
error Command failed with exit code 1.

**Comment by amyegan** [comment]
Hi @lucastey! Do you have a public repo or minimal reproduction that you can share?

**Reply by lucastey** [reply]
Hey! The repo is actually a clone of the Portfolio Starter Kit by vercel. Apart from changing some of the static text for personalization, the base template remains pretty much as it is. Is it better if I add you as a collaborator into my repo?

**Reply by lucastey** [reply, marked as answer]
Resolved!

Issue was with nextjs+react version incompatibilities

Upgraded next to >v13 and react to v18 resolved the Type/webpack errors
